modules/ml/include/opencv2/ml.hpp

```diff
@@ -790,7 +790,7 @@ class CV_EXPORTS_W EM : public StatModel
     Returns vector of covariation matrices. Number of matrices is the number of gaussian mixtures,
     each matrix is a square floating-point matrix NxN, where N is the space dimensionality.
      */
-    virtual void getCovs(std::vector<Mat>& covs) const = 0;
+    CV_WRAP virtual void getCovs(std::vector<Mat>& covs) const = 0;
```

Comment:
Shouldn't it be `CV_OUT std::vector<Mat>& covs` to use it as expected: `covs = mdl.getCovs()`?